### DISCUSSION TITLE
Multiple language for gotenberg

### DISCUSSION THREAD

**Discussion by aviellg** [gotenberg/gotenberg#629] [Q&A]
i am using Gotenberg with paperless-ngx to  convert docx files to pdf 
I am in need to convert english ( LTR LANGUAGE)  and hebrew files (RTL language )
after google search seems i need to create my own container as gotenberg does not support it by default  .
does anyone have a guide me how we do it ? preferably video

**Comment by gulien** [comment]
See: https://gotenberg.dev/docs/modules/libreoffice#switch-language

**Reply by 390620652** [reply]
how to switch language to zh-cn ?

**Reply by gulien** [reply]
Hello @390620652,

The documentation URL is now: https://gotenberg.dev/docs/configuration#switch-language
